easybuild/easyblocks/p/pgi.py

```diff
@@ -0,0 +1,94 @@
+##
+# Copyright 2015 Bart Oldeman
```

Comment:
is it ok to use the same header as used in easyblocks/c/clang.py? (as far as I can see in CONTRIBUTING.md this is ok)

# This file is triple-licensed under GPLv2 (see below), MIT, and
# BSD three-clause licenses.